tools/scylla-nodetool.cc

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2023-present ScyllaDB
+ */
+
+/*
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+#include <seastar/core/thread.hh>
+
+#include "log.hh"
+#include "tools/utils.hh"
+
+namespace bpo = boost::program_options;
+
+namespace {
+
+const auto app_name = "scylla-nodetool";
+
+logging::logger nodetool_log(app_name);
```

Comment:
nit, s/nodetool_log/logger/. as, in this context, the only logger is supposed to be used is `log`. no need to differentiate it from other `logger`.

Reply:
`log` name doesn't work, conflicts with some macro from libc. Used `nlog` instead.